cypress/integration/Login tests/login.specs.js

```diff
@@ -0,0 +1,72 @@
+ /// <reference types="cypress" />
+ 
+ describe("NeetoAuth Login Test", () => {
+   let userDetails;
+   
+   beforeEach(() => {
+     cy.viewport(1280,720);
+     cy.visit('https://spinkart.neetoauth.net')
+     cy.fixture("credentials").then((user)=>{
+       userDetails = user;
+     })
+  });
+    
+  it("Login test with correct email and correct password", () => {
+      cy.login(userDetails.correct.email,userDetails.correct.password);
+      cy.loginSubmit();
+  });
+
+  it("Login test with correct email and wrong password", () => {
+      cy.login(userDetails.correct.email,userDetails.incorrect.password);
+      cy.loginSubmit();
+      cy.msgPrompt('Something went wrong.');
+  });
+
+  it("Login test with wrong email and correct password", () => {
+      cy.login(userDetails.incorrect.email,userDetails.correct.password);
+      cy.loginSubmit();
+      cy.msgPrompt('Something went wrong.');
+  });
+
+  it("Login test with wrong email and wrong password", () => {
+      cy.login(userDetails.incorrect.email,userDetails.incorrect.password);
+      cy.loginSubmit();
+      cy.msgPrompt('Something went wrong.');
+  });
+
+  it("Login test with empty email and empty password", () => {
+      cy.get('[data-cy="login-email-text-field"]').and(($input) => {
+            expect($input).to.have.value('')
+      })
+      cy.get('[data-cy="login-password-text-field"]').and(($input) => {
+            expect($input).to.have.value('')
+      })
+      cy.loginSubmit();
+      cy.location().should((loc) => {
```

Comment:
There should be logical separation(empty line) between commands. Please read the cypress guidelines for better idea.
```suggestion
      cy.loginSubmit();
      
      cy.location().should((loc) => {
```